md/bdm-access-control.md

```diff
@@ -5,8 +5,7 @@
 :::
 
 ## Overview
-Currently, in Bonita, defining access control on a business object (BO) means defining which attributes of this business object a given user is allowed to access through REST API.  
-If access controls are defined on the BDM, when a user calls a BDM API to retrieve some objects, the response will only contain the attributes this user is allowed to see.
+This page explains how to protect the access to your BDM. It explores all the methods available in Bonita, both legacy methods (through [Rest api authorizations](rest-api-authorization.md)), and new 7.7 Access Control feature.
```

Comment:
Not sure it is clearer this way, but I let others decide...

Reply:
agree, i would do something like

-Currently, in Bonita, defining access control on a business object (BO) means defining which attributes of this business object a given user is allowed to access through REST API.  The access control feature doesn't give the possibility to define which instance of a business object a user is allowed to access. It is still possible to do so using Bonita, see exemple 3.  
 -If access controls are defined on the BDM, when a user calls a BDM API to retrieve some objects, the response will only contain the attributes this user is allowed to see.

Reply:
I will let nath decide on this one.

Reply:
This page explains how to protect the access to your BDM through REST API calls. 
The BDM access control feature, available in Bonita Studio, allows to grant access to entire business objects or access to some attributes of business objects.
To grant access to specific instances of business objects, you will need to use [Rest api authorizations](rest-api-authorization.md)).
This page explains how to use both.